index.js

```diff
@@ -1,25 +1,4 @@
-var RCTAddressBook = require('react-native').NativeModules.AddressBook
-var invariant = require('invariant')
-var logError = require('logError')
-var warning = require('warning')
+var React = require('react-native');
+var AddressBook = React.NativeModules.AddressBook;
 
-var AddressBook = {
-
-  getContacts(callback) {
-    RCTAddressBook.getContacts(callback)
-  },
-
-  addContact(record, callback) {
-    RCTAddressBook.addContact(record, callback)
-  },
-
-  checkPermission(callback) {
-    RCTAddressBook.checkPermission(callback)
-  },
-
-  requestPermission(callback) {
-    RCTAddressBook.requestPermission(callback)
-  }
-}
-
-module.exports = AddressBook
+module.exports = AddressBook;
```

Comment:
This could be `module.exports = require('react-native').NativeModules.AddressBook` as a one liner. The rest of the readme was great